### PR TITLE
Set up Travis CI

### LINKED ISSUE #3: Add Continous Integration testing.
Also, investigate `xctool` hang on `testZapAllStyleParameters`.

### PR DESCRIPTION
**Pull request by ikesyo**
Related to https://github.com/jdhealy/PrettyColors/pull/12#issuecomment-231184616. Resolves #3.

### CODE REVIEW

**Comment by ikesyo** [comment]
CI passed on my fork: https://travis-ci.org/ikesyo/PrettyColors/builds/147183453

**Comment by ikesyo** [comment]
https://github.com/jdhealy/PrettyColors/issues/3#issue-53962934: `xctool` won't support Xcode 8 and will be deprecated, so this does not test with the tool. See https://github.com/facebook/xctool/issues/704 for the details.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        - JOB=CARTHAGE
+      script:
+        - brew update
+        - brew outdated carthage || brew upgrade carthage
+        - carthage build --no-skip-current
+        - for platform in Mac iOS tvOS watchOS; do test -d Carthage/Build/$platform/PrettyColors.framework || exit 1; done
+git:
+  submodules: false
+before_install:
+  - git submodule update --init --recursive
```
Comment (jdhealy) [inline .travis.yml:66]: Curious about the above lines, assuming they're a workaround for some issue?

I wouldn't be concerned about them, but jobs are failing with `fatal: unable to access 'https://github.com/jdhealy/FABConfig.git/': Failed to connect to github.com port 443: Operation timed out`: https://travis-ci.org/jdhealy/PrettyColors/jobs/148949181#L281 — possibly getting throttled by GitHub?

**Comment by jdhealy** [comment]
Overall fantastic — thanks so much!

Made a few follow up commits: [(1)](https://github.com/jdhealy/PrettyColors/pull/12/commits/8033ea24f4142c67fd6a770b8f1fb342d1226faa) [(2)](https://github.com/jdhealy/PrettyColors/pull/12/commits/6ae260feed5d3df659a6a15950805f25f9a5bcdf) [(3)](https://github.com/jdhealy/PrettyColors/pull/12/commits/c78faeb779b3ce200384b7ebb1e0d5c8f345e802) [(4)](https://github.com/jdhealy/PrettyColors/pull/12/commits/cee577576725b0e932727b2e635f3b41e41fe8ad)

[One question about submodule initialization](https://github.com/jdhealy/PrettyColors/pull/13/files#r73025045), though. 😄

**Pull request merged**